tests/test_0173_empty_and_multiprocessing_bugs.py

```diff
@@ -17,8 +16,8 @@ def test_empty():
         assert t["z"].array(library="np").tolist() == []
 
 
-def readone(filename):
-    with uproot.open(filename) as f:
+def read_one(filename):
+    with uproot.open(filename, handler=uproot.source.file.MemmapSource) as f:
```

Comment:
https://github.com/scikit-hep/uproot5/issues/173

This won't work with fsspec at the moment (meaning it won't work without manually specifing a different source), I'm not sure what the consecuences might be

Reply:
From the looks of it, this test was going after a bug that's particular to memory-mapping. Explicitly setting the handler to recreate that situation is fine.